packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
Take a look here:https://github.com/RocketChat/Rocket.Chat/blob/d327724c4b50f23af36e75cafbbae546eb322dd6/client/startup/startup.js#L76 I'm not sure why these two approaches differ